src/ui/Product.svelte

```diff
@@ -1,21 +1,45 @@
 <script lang="ts">
+  import Svg from '@/ui/Svg'
+
   let className = ''
   export { className as class }
   export let title: string
+  export let isHalloween: string
 </script>
 
-<a href="/" class="body-2 txt-m title row v-center {className}" on:click={window.__onLinkClick}>
-  <img src="{process.env.MEDIA_PATH}/illus/san-logo.svg" alt="SAN" class="mrg-s mrg--r" />
-  {title}
-</a>
+<div class="relative">
+  {#if isHalloween}
+    <Svg illus id="batman" w="57" h="52" class="$style.batman" />
+  {/if}
```

Comment:
`div` is redundant. Render inside `a`

Reply:
It has to be due to positioning bat behind text and logo